### PR TITLE
Support slash characters in testcase names

### PR DESCRIPTION
**Pull request by hluk**
JIRA: RHELWF-7012

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -11,3 +11,4 @@ Flask >= 0.10.1
 iso8601 >= 0.1.11
 resultsdb_api >= 2.0
 six >= 1.10.0
+cachelib
```
Comment (chuazhan) [inline requirements.txt:14]: Why this is needed?

**Review by chuazhan — COMMENTED** [review]
Not sure why the cachelib is needed?

File: requirements.txt (hunk 1 above)
Comment (hluk) [reply requirements.txt:14]: It is used in `resultsdb_frontend/controllers/main.py`. My guess is that previously it was installed as a dependency of a library mentioned in `requirements.txt` transitively.

**Review by chuazhan — APPROVED** [review]
+1

**Pull request merged**